### PR TITLE
Issue #352: Polish dataflows list page.

### LINKED ISSUE #352: Polish overview page
* [x] remove configuration column
* [x] remove steps count column
* [x] rename user column to what it is, ie Created by, or Last mod by
* [x] make the preview image have a maximum width and max height and let it auto scale
* [x] have a 'Last run' column with the human readable time since last run, eg 2 hours 3 mins ago

### PR DESCRIPTION
**Pull request by jaypha**
Resolves #352

### CODE REVIEW

**Comment by jaypha** [comment]
Screenshot from 2022-08-02 17-36-46

File: classes/dataflows_table.php [hunk 1]
```diff
@@ -91,7 +97,14 @@ public function col_preview(\stdClass $record): string {
         global $OUTPUT;
         $imgurl = new \moodle_url('/admin/tool/dataflows/visual.php', ['id' => $record->id, 'type' => 'png']);
         if (helper::is_graphviz_dot_installed()) {
-            $img = \html_writer::img($imgurl, "Dataflow #{$record->id} visualisation", ['height' => '30px', 'class' => 'lightbox']);
+            $img = \html_writer::img(
+                $imgurl,
+                "Dataflow #{$record->id} visualisation",
```
Comment (brendanheywood) [inline classes/dataflows_table.php:102]: lang string please (even though it was like this before)

Comment (brendanheywood) [reply classes/dataflows_table.php:102]: Or this could be alt="" which is also fine

Comment (brendanheywood) [reply classes/dataflows_table.php:102]: I've merged without it and done this already

**Pull request merged**